lib/index.js

```diff
@@ -54,15 +55,20 @@ class Insight {
 		this.config.set('clientId', val);
 	}
 	_save() {
-		// Debounce in case of rapid .track() invocations
-		debounce(() => {
-			const cp = fork(path.join(__dirname, 'push.js'), {silent: true});
-			cp.send(this._getPayload());
-			cp.unref();
-			cp.disconnect();
-
-			this._queue = {};
-		}, 100)();
+		process.nextTick(() => {
```

Comment:
I think you want to use `setImmediate` here.

Reply:
Good call, will change it.